Example/CRToastDemo.xcodeproj/project.pbxproj

```diff
@@ -380,7 +384,7 @@
 				GCC_PREFIX_HEADER = "CRToastDemo/CRToastDemo-Prefix.pch";
 				GCC_WARN_PEDANTIC = YES;
 				INFOPLIST_FILE = "CRToastDemo/CRToastDemo-Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.2;
```

Comment:
Does this change the build/deployment target to 11.2? Is there any technical reason to change the build target to 11.2?